test/distributed/cases/statement_query_type/statement_query_type.result

```diff
@@ -257,26 +253,19 @@ show create database test_db    Other    external_sql
 create view test_view as select * from test_table    DDL    external_sql
 create table test_table(\ncol1 int,\ncol2 varchar\n)    DDL    external_sql
 use test_db    Other    external_sql
-create database test_db    DDL    external_sql
 begin    TCL    external_sql
+create database test_db    DDL    external_sql
```

Comment:
i confuse. why this stmt changes the order ?

Reply:
Becasue the aggr,  some stmts, the one that should be aggreagted, will be put into a window first.  And will be written down at that end. 

So the order will be modified.

Reply:
i aware this rule.
In here, this two sql is  `begin` and `create`, both are go into  window.  is that right ?

Reply:
my fail. current result is the corect one.